src/opentimelineio/anyDictionary.h

```diff
@@ -122,6 +123,53 @@ class AnyDictionary : private std::map<std::string, any>
         map::swap(other);
     }
 
+    /// @TODO: need the right policy for accessing/returning membors
+    template<typename containedType>
+    containedType
+    get_default(
```

Comment:
-> `get_or_default()`

Reply:
switched it around to `get_if_set` but open to suggestions on this name (if we keep it, which -- also borderline on that)